tests/test_client.py

```diff
@@ -0,0 +1,117 @@
+"""
+Tests for the client
+"""
+
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import unittest
+import mock
```

Comment:
The convention I've adopted so far with imports is to have a number of sections. (1) futures; (2) standard libary; (3) third party libraries; (4) package local. Since mock isn't part of that standard library until Python 3.3, which we're not supporting yet, so we should put this into third party stuff. Also, I think we'll need to update the Travis script to install mock for testing.